src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+const largeImg = gallery.querySelector('.gallery__large-img');
+
+// eslint-disable-next-line no-shadow
+gallery.addEventListener('click', (event) => {
+  event.preventDefault();
```

Comment:
Consider whether `event.preventDefault()` is necessary here. If the default action of the link (e.g., navigating to a new page) is not causing any issues, you might not need to prevent it. Ensure that preventing the default action is required for your use case.